src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -123,9 +123,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     private function searchForEvent(EventDispatcherInterface $dispatcher, $needle): array
     {
         $output = [];
+        $lcNeedle = strtolower($needle);
         $allEvents = array_keys($dispatcher->getListeners());
         foreach ($allEvents as $event) {
-            if (str_contains($event, $needle)) {
+            if (str_contains(strtolower($event), $lcNeedle)) {
```

Comment:
We could use `false !== stripos($event, $needle)`. But Im not sure what is quicker.

Reply:
I'd say performance difference is not important here because this is a console command for the dev environment. It's not something with impact in prod.

Reply:
Except that  `str_contains` is PHP 8 only, let's use what @Nyholm suggested instead.